spec/requests/providers/home_address/lookups_controller_spec.rb

```diff
@@ -17,13 +17,32 @@
     context "when the provider is authenticated" do
       before do
         login_as provider
+        applicant
         get_request
       end
 
       it "shows the postcode entry page" do
         expect(response).to be_successful
         expect(unescaped_response_body).to include("Find your client's home address")
       end
+
+      context "when the provider correspondence_address_choice is home" do
+        let(:applicant) { create(:applicant, legal_aid_application:, correspondence_address_choice:) }
```

Comment:
Could this be a one-liner?
```ruby 
let(:applicant) { create(:applicant, legal_aid_application:, correspondence_address_choice: "home")
```
As setting the correspondence_address_choice is not overridden at a higher level